### PR TITLE
Fix message in error suggestion when only one suggestion is available

### PR DESCRIPTION
**Pull request by eksperimental**

### CODE REVIEW

**Comment by josevalim** [comment]
The other option is to pick a choice of words that will work for one or many. Any ideas?

File: lib/elixir/test/elixir/exception_test.exs [hunk 1]
```diff
@@ -517,7 +517,7 @@ defmodule ExceptionTest do
       message = blame_message(%{first: nil, second: nil}, fn map -> map.firts end)
 
       assert message == """
-             key :firts not found in: %{first: nil, second: nil}. Did you mean one of:
+             key :firts not found in: %{first: nil, second: nil}. Did you mean:
```
Comment (kelvinst) [inline lib/elixir/test/elixir/exception_test.exs:520]: Little typo:

```suggestion
             key :first not found in: %{first: nil, second: nil}. Did you mean:
```

File: lib/elixir/test/elixir/exception_test.exs [hunk 2]
```diff
@@ -535,7 +535,7 @@ defmodule ExceptionTest do
         blame_message([first: nil, second: nil], fn kwlist -> Keyword.fetch!(kwlist, :firts) end)
 
       assert message == """
-             key :firts not found in: [first: nil, second: nil]. Did you mean one of:
+             key :firts not found in: [first: nil, second: nil]. Did you mean:
```
Comment (kelvinst) [inline lib/elixir/test/elixir/exception_test.exs:538]: Little typo:

```suggestion
             key :first not found in: %{first: nil, second: nil}. Did you mean:
```

**Review by kelvinst — APPROVED** [review]
Found some typos on the messages you touched 😬 

EDIT: nevermind, saw it was actually a test 🤦

**Comment by kelvinst** [comment]
> The other option is to pick a choice of words that will work for one or many. Any ideas?

I think "Did you mean:" would be good enough for that purpose. I mean, I totally don't care for the total semantical correctness of the suggestion message anyway, so I would stick to the shorter version.

**Comment by eksperimental** [comment]
Well. Here' s the original issue that I faced (this PR was a partial fix to it)

```console
Compiling 1 file (.ex)
warning: Hello.Orders.get_order!/1 is undefined or private. Did you mean one of:

      * get_order!/2

Found at 3 locations:
  lib/hello_web/controllers/order_controller.ex:37: HelloWeb.OrderController.edit/2
  lib/hello_web/controllers/order_controller.ex:43: HelloWeb.OrderController.update/2
  lib/hello_web/controllers/order_controller.ex:57: HelloWeb.OrderController.delete/2
```

* this warning is not clear. It is ambiguous and it could be interpreted that the available function is found in these 3 locations, nor the undefined function (that is what I actually first crossed my mind)
* Also when there is only one available the suggestion should read, "Did you mean:"

A much clear message should read:

```console
Compiling 1 file (.ex)
warning: Hello.Orders.get_order!/1 is undefined or private. Did you mean:

      * get_order!/2

Hello.Orders.get_order!/1 call found at 3 locations:
  lib/hello_web/controllers/order_controller.ex:37: HelloWeb.OrderController.edit/2
  lib/hello_web/controllers/order_controller.ex:43: HelloWeb.OrderController.update/2
  lib/hello_web/controllers/order_controller.ex:57: HelloWeb.OrderController.delete/2
```

The thing is that modifying the "found at 3 locations" part is not as simple as replacing a string.

**Comment by eksperimental** [comment]
> > The other option is to pick a choice of words that will work for one or many. Any ideas?
> 
> I think "Did you mean:" would be good enough for that purpose. I mean, I totally don't care for the total semantical correctness of the suggestion message anyway, so I would stick to the shorter version.

I agree. "Did you mean" is good enough for one or more suggestions if we only want to keep one message.

**Comment by josevalim** [comment]
So let's remove one of. :)

**Comment by eksperimental** [comment]
> So let's remove one of. :)

what about the "found at 3 locations" part.
If i read it in plain, I understand that the suggestion is found at 3 location, not the original string.

**Comment by josevalim** [comment]
we can say "Invalid call found at 3 locations: "

**Comment by eksperimental** [comment]
> we can say "Invalid call found at 3 locations: "

I will submit a second PR to fix this.

**Comment by eksperimental** [comment]
> So let's remove one of. :)

done

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Comment by eksperimental** [comment]
Thank you @kelvinst for the review and comments

**Pull request merged**